devtasks.py

```diff
@@ -42,7 +42,7 @@ def dev_setup():
         local["poetry"]("install")
 
 
-def lint(recycle_container=False):
+def lint(_recycle_container=False):
```

Comment:
Since this argument is unused, we should remove it instead of renaming it:

```suggestion
def lint():
```

Reply:
done